Cron/Order.php

```diff
@@ -52,6 +55,12 @@ class Order
      */
     const ORDER_FETCH_SLEEP_TIME = 2;
 
+     /**
+     * Constants representing keys for store queue-access components
+     */
+    const QUEUE_URL_KEY = 'url';
+    const HTTP_CLIENT_KEY = 'httpClient';
```

Comment:
I think each constant should get its own docblock, in case we ever start using PHPDoc to generate documentation.

Reply:
👍